tests/rptest/services/redpanda.py

```diff
@@ -2870,7 +2870,12 @@ def remove_local_data(self, node):
 
     def redpanda_pid(self, node):
         try:
-            cmd = "ps ax | grep -i 'redpanda' | grep -v grep | grep -v 'version'| grep -v \"\[redpanda\]\" | awk '{print $1}'"
+            cmd = " | ".join([
+                "ps ax", "grep -i redpanda", "grep -v grep", "grep -v version",
```

Comment:
I wonder if it's time to move to `pgrep redpanda`

Reply:
Yeah it'd be simpler - is `pgrep` installed in the all the environments we run ducktape in?

Reply:
I just checked locally and it's installed. I can kick off a run of cdt to double check.